modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -5,7 +5,9 @@ This section highlights new features in {product} {product-version}.
 
 == {product} Operator is now generally available (GA)
 
-You can use the {product} Operator to install {product-short} on your {ocp-short}cluster. For more information, see the link:{LinkAdminGuide}[Installing {product} using the Operator] section in the _Administration guide_.
+You can use the {product} Operator to install {product-short} on your {ocp-short} cluster.
+//For more information, see the link:{LinkAdminGuide}[Installing {product} using the Operator] section in the _Administration guide_.
+//replace with link to the installation guide.
```

Comment:
this info is no longer in the Admin guide; replace with a link to the proper title in a separate PR